timesketch/lib/datastores/opensearch.py

```diff
@@ -998,7 +996,7 @@ def import_event(
             self.import_events.append(event)
             self.import_counter["events"] += 1
 
-            if self.import_counter["events"] % int(flush_interval) == 0:
+            if self.import_counter["events"] % int(self.flush_interval) == 0:
```

Comment:
Have you checked so we don't use this argument anywhere? E.g. in the api client.

Reply:
5ac3397584474fc1b5b934e0d8defd90bb9d641b reverts the argument